dom/ranges/Range-mutations.html

```diff
@@ -86,12 +86,17 @@
     ]);
     tests.push([
       descFn(params) + ", with selected " + describeRange(params[len - 4], params[len - 3], params[len - 2], params[len - 1]),
-      function(params) { return function() {
+      function(params) { return function(selectedRange) {
         var evaledParams = params.map(eval);
         for (var i = 0; i < evaledParams.length; i++) {
           assert_true(typeof evaledParams[i] != "undefined",
             "Test bug: " + params[i] + " is undefined");
         }
+        // Override input range with the one that was actually selected when computing the expected result.
```

Comment:
Whoops, I should have noticed that.  My bad, sorry.  I like the new slice/concat way of doing things.  :)